File: EquipmentApi/EquipmentApi/Dtos/ReadDtos/EquipmentDto.cs
using System;

namespace EquipmentApi.Dtos.ReadDtos
{
    public class EquipmentDto
    {
        public Guid Id { get; set; }
        public string Name { get; set; }

        public EquipmentModelShowDto EquipmentModel { get; set; }
    }
}

File: forestequipment/src/main/java/com/vitor/forestequipment/dtos/EquipmentDto.java
package com.vitor.forestequipment.dtos;

import java.util.UUID;

public class EquipmentDto {

	private UUID id;

	private String name;

	private UUID model;

	public String getName() {
		return name;
	}

	public void setName(String name) {
		this.name = name;
	}

	public UUID getModel() {
		return model;
	}

	public void setModel(UUID equipmentModel) {
		this.model = equipmentModel;
	}

	public UUID getId() {
		return id;
	}

	public void setId(UUID id) {
		this.id = id;
	}

}
